.github/workflows/ccpp.yml

```
@@ -35,18 +43,12 @@ jobs:
-            os: ubuntu-latest
-            toolchain: ninja-gcc-cxx17-fpic
-          - name: macos-clang-hunter
-            os: macos-latest
-            toolchain: ninja-clang-cxx17-fpic
-          - name: windows-msvc-hunter
-            os: windows-latest
-            toolchain: ninja-vs-win64-cxx17
 
     steps:
-    - uses: actions/checkout@v3
+    - name: ccache
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)